numpy/random/_mt19937.pyx

```diff
@@ -226,6 +226,23 @@ cdef class MT19937(BitGenerator):
         -------
         bit_generator : MT19937
             New instance of generator jumped iter times
+
+        Notes
+        -----
+        The jump step is computed using a modified version of Matsumoto's
+        implementation of Horner's method. The step polynomial is precomputed
+        to perform 2**128 steps. The jumped state has been verified to match
+        the state produced using Matsumoto's original code.
+
+        References
+        ----------
+        .. [1] Matsumoto, M, Generating multiple disjoint streams of
+           pseudorandom number sequences.  Accessed on: May 6, 2020.
```

Comment:
I suspect that the the title should be in quotes, but I don't know if that makes a difference. I'm not familiar with the citation format we use. @mattip @rkern thoughts?

Reply:
This may be related to regexp for renaming references in numpydoc... looking into it now

Reply:
hmm doesn't seem like it, at least not a problem with the section I looked at: 

https://github.com/numpy/numpydoc/blob/b4c5fd17e2b85c2416a5e586933eee353b58bf7c/numpydoc/numpydoc.py#L43-L67